packages/kbn-test/src/functional_tests/cli/start_servers/cli.js

```diff
@@ -16,8 +17,19 @@ import { processOptions, displayHelp } from './args';
  *                                   if no config option is passed
  */
 export async function startServersCli(defaultConfigPath) {
+  const runStartTime = Date.now();
+  const log = new ToolingLog({
+    level: 'info',
+    writeTo: process.stdout,
+  });
+  const reportTime = getTimeReporter(log, 'functional_tests_server');
+
   await runCli(displayHelp, async (userOptions) => {
     const options = processOptions(userOptions, defaultConfigPath);
-    await startServers(options);
+    await startServers({
+      ...options,
+      runStartTime,
+      reportTime,
```

Comment:
Not sure why need to pass `eportTime` here, and not just create it in startServers.